docsrc/README.md

```diff
@@ -0,0 +1,22 @@
+sphinx
+======
+
+## Getting started - building from source.
+1. Follow the `Project Setup` from the Readme.mp from the root of the project. 
```

Comment:
what do you mean by "building from source" here...are you referring to loading the conda environment, or building something related to sphinx?